com.unity.render-pipelines.universal/Runtime/Passes/AdditionalLightsShadowCasterPass.cs

```diff
@@ -852,9 +852,8 @@ void Clear()
             m_GlobalShadowSliceIndexToPerLightShadowSliceIndex.Clear();
         }
 
-        void SetEmptyAdditionalShadowmapAtlas(ref ScriptableRenderContext context)
+        void SetEmptyAdditionalShadowmapAtlas(ref ScriptableRenderContext context, CommandBuffer cmd)
```

Comment:
should we pass RenderingData instead of cmd?

Reply:
fixed in 3939959